### PR TITLE
use variables for paywall colors. fix overflow

### LINKED ISSUE #4489: paywall needs to use variables and overflow is not working properly
image 1:
Screen Shot 2020-07-10 at 11 11 12 AM

image 2:
Screen Shot 2020-07-10 at 11 11 19 AM

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4489 

Crosses one item in #4525 

**Description:**

Good to go! :woman_dancing: 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f0c6bfe30a4b806346efe4d--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
Rerunning E2E, but this is a very easy PR. Let's merge! :woman_dancing:

File: app/src/router.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ export const routeGuard = (store) => async (to, from, next) => {
     return
   }
   if (to.name === `notifications`) {
+    // firebase requires some time to sign the user in after the app is initialized
+    await new Promise((resolve) => setTimeout(resolve, 1000))
```
Comment (jbibla) [inline app/src/router.js:21]: uhhhhh this seems funny

Comment (Bitcoinera) [reply app/src/router.js:21]: :laughing:

Comment (Bitcoinera) [reply app/src/router.js:21]: it is because the Firebase observer takes a while to get sign in state

**Comment by faboweb** [comment]
tests fail

**Comment by Bitcoinera** [comment]
Let's merge! :woman_dancing:

**Pull request merged**